### PR TITLE
5773 - Column Chart Rotate Option

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR will add a rotate option in Column Chart.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/5773

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/column/example-rotate.html

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — COMMENTED** [review]
Looks good except for the tests. Also for the example maybe just show slightly shorted labels as it sort of takes up the whole bottom of the chart so maybe isnt the best to show it this way.

**Review by ericangeles — APPROVED** [review]
Nice work!! 🍌

File: app/views/components/column/example-rotate.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+<div class="row">
+  <div class="two-thirds column">
+      <div class="widget">
+        <div class="widget-header">
+          <h2 class="widget-title">Column Chart Example - xAxis Two lines</h2>
```
Comment (tmcconechy) [inline app/views/components/column/example-rotate.html:6]:
```suggestion
          <h2 class="widget-title">Column Chart Example - xAxis Rotated</h2>
```

**Review by tmcconechy — APPROVED** [review]
Just changed the title otherwise looks muy bueno

**Review by ericangeles — APPROVED** [review]

**Pull request merged**